Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1111,31 +1035,42 @@
     <RegEx find="([oO])pšt(a|e|i|u|em|om|im)" replaceWith="$1pć$2" />
     <RegEx find="([oO])rganizuje(m|mo|š|te)?" replaceWith="$1rganizira$2" />
     <RegEx find="([oO])rganizov" replaceWith="$1rganizir" />
+    <RegEx find="([oO])ružij[aeu]" replaceWith="$1ružj" />
     <RegEx find="([oO])seti(o|m|š|la|li|le|ti)" replaceWith="$1sjeti$2" />
     <RegEx find="([oO])setljiv(a|i|e|o|om|im|ima|oj)?" replaceWith="$1sjetljiv$2" />
     <RegEx find="([oO])sj?e[ćč]a" replaceWith="$1sjeća" />
     <RegEx find="([oO])strv(a|u|om)" replaceWith="$1tok$2" />
     <RegEx find="([oO])sveženj(a|e|u|em|ima)" replaceWith="$1svježenj$2" />
     <RegEx find="([oO])svežava(m|š|mo|te|ju|ti|li|la|lo|ju|juć|juće|juća)?" replaceWith="$1svježava$2" />
-    <RegEx find="\b([oO])ter(a|am|amo|ate|aju)" replaceWith="$1tjer$2" />
+    <RegEx find="\b([oO])tera(m|mo|te|ju)?" replaceWith="$1tjera$2" />
```

Comment:
I think, there might be a `\b` missing at the end of the regular expression:
```
<RegEx find="\b([oO])tera(m|mo|te|ju)?\b" replaceWith="$1tjera$2" />
```
Unless you really want to replace every "otera" and "Otera" at the start of a word:
```
<RegEx find="\b([oO])tera" replaceWith="$1tjera" />
```
(the *optional* `(m|mo|te|ju)` group serves no purpose)